src/components/card-view/card-view-view.js

```diff
@@ -0,0 +1,361 @@
+        }
+
+        .green {
+          background: #D3EA98;
+        }
+
+        .dark-green {
+          color: #4CAF50;
+        }
+
+        .orange {
+          color: #FF7849;
+        }
+
+        .red {
+          color: #C22727;
+        }
+
+        .pink {
+          background: #FFC8BF;
+        }
+
+        .priority  {
+          font-size: 12px;
+          color: #212121;
+        }
+
+        .status-background {
+          border-radius: 2px;
+          padding: 4px;
+        }
+
+        .picture {
+          cursor: pointer;
+        }
+
+        .status {
+          font-size: 12px;
+          color: #FFFFFF;
+        }
+
+        .dark-brown {
+          background: #C22727;
+        }
+
+        .border {
+          width: 100%;
+          background: rgba(115, 143, 179, 0.3);
+          height: 1px;
+          margin-bottom: 12px;
+        }
+
+        .name {
+          font-weight: 500;
+          font-size: 14px;
+          line-height: 24px;
+          color: #616161;
+          margin-left: 10px;
+        }
+
+        .wrapper {
+          display: flex;
+          align-items: space-between;
+        }
+
+        .donut-wrapper {
+          width: 180px;
+          height: auto;
+          position: relative;
+          right: 0;
+        }
+
+    `;
+  }
+
+  static get properties() {
+    return {
+      /**
+       * The header of the component.
+       */
+      title: {type: String},
+
+      /**
+       * Funtion to run when more icon is clicked.
+       */
+      moreIconClicked: {type: Function},
+
+      /**
+       * Function to run when clicked on profile.
+       */
+      profileClicked: { type: Function },
+
+      /**
+       * Boolean whether to show or hide status.
+       */
+      showStatus: { type: Boolean }
+    };
+  }
+
+  constructor() {
+    super();
+
+    this.isBig = true;
+    this.isMedium = false;
+    this.isSmall = false;
+    this.showStatus = true;
+
+    this.title = 'Card Header';
+    this.picture = AVATAR;
+    this.progressValue = 80;
+    this.progressColor = 'green';
+    this.dueDate = '2020-20-20';
+    this.startedDate = '2020-20-20';
+    this.priority = 'Medium';
+    this.status = 'Blocked';
+    this.statusColor = 'dark-brown';
+    this.priorityColor =  'yellow';
+    this.indicatorColor = 'dark-green';
+    this.username = 'Harry Potter';
+    this.estimatedDuration = 100000;
+    this.actualDuration = 200000;
+
+    this.chartValue = [['this', 'that'], ['Done', 100000], ['Remaining', 2000000]];
+    this.chartColors = ['red', '#EEEEEE'];
+  }
+
+  render() {
+    return html`
+      <paper-card>
+        <div class="header">
+            <div class="title">
+                ${this.title}
+            </div>
+            <div>
+              <paper-icon-button icon="icons:more-vert" class="more-icon" @click="${this.moreIconClicked}"></paper-icon-button>
+            </div>
+        </div>
+    ${this.isBig? this.renderBigCard(): nothing}
```

Comment:
use switch case instead of boolean values. Give the value cardSize from parent and use switch case to render cards